fl4health/utils/losses.py

```diff
@@ -2,12 +2,47 @@
 
```

Comment:
I think maybe we want to make an explicit EvalLosses class that inherits from Losses, similar to TrainLosses to force and explicit "fork." One thing that I think can be in EvalLosses but not in TrainLosses is checkpoint. It's possible that a person might want to checkpoint in the training loop, but they can do that explicitly through additional_losses if they really want. I think checkpointing during validation is what makes the most sense to support by default.